docs/chapters/baseplatform/deploying-and-booting-the-image.rst

```diff
@@ -0,0 +1,64 @@
+Writing the image
+=================
+
+This section describes how to write the build output, a ready disk-image, to a removable media. Assuming you are positioned in the PELUX top-level Yocto directory follow the instructions stated below. Further information can be found in the `Poky documentation`_.
+
+NUC / Minnowboard
+-----------------
+
+There are three different methods of writing an Intel ``.hddimg`` or ``.wic`` image to a USB-stick.
+
+1. Use the ``dd`` utility to write the image to the raw block device. For example:
+
+.. code-block:: bash
+
+    dd if=build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg \
+    of=<host-device> bs=4M
+
+2. Poky provides a convenience script for writing images. Using an unmounted USB-stick or SD-card, do:
+
+.. code-block:: bash
+
+    sources/poky/scripts/contrib/mkefidisk.sh <host-device> \
+    build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg \
+    <target-device>
+
+3. PELUX provides a wrapper around the poky convenience script for writing images. It decreases the write time to the removable media:
+
+.. code-block:: bash
+
+    sources/meta-pelux/scripts/mkefi_wrap \
+    poky/scripts/contrib/mkefidisk.sh \
+    build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg \
+    <host-device>
+
+The ``<host-device>`` is the device on the host, e.g. ``/dev/sdc``, and ``<target-device>`` is how the target hardware will see the media, e.g. ``/dev/sda``.
+
+Raspberry Pi 3
+--------------
+
+Write the generated image file to an SD-card using e.g.:
+
+.. code-block:: bash
+
+    dd if=build/tmp/deploy/images/raspberrypi3/core-image-pelux-raspberrypi3.rpi-sdimg \
```

Comment:
Same here perhaps just mention where to find it in the build directory but then I'd do `dd if=ore-image-pelux-raspberrypi3.rpi-sdimg ...`